docs/features/keyboard-support.md

```diff
@@ -195,6 +195,15 @@ Use the following keystrokes for more efficient navigation in the CKEditor&nbsp;
 }
 </style>
 
+## Displaying keyboard shortcuts in the editor
```

Comment:
Did you have a chance to check if it is feasible to display the a11y dialog contents automatically, instead of the table above? I think it was a cool idea.

Reply:
Yes, I considered that. This would butcher SEO, though as the editor must run first for the content to generate. So I figured out duplication is safer.